webui/src/lib/components/repository/changes.jsx

```diff
@@ -145,3 +151,103 @@ function useTreeItemType(entry, repo, leftDiffRefID, rightDiffRefID) {
+ * @param onRevert to be called when an object/prefix is requested to be reverted
+ */
+export const ChangesTreeContainer = ({results, showExperimentalDeltaDiffButton = false, delimiter, uriNavigator,
+                                         leftDiffRefID, rightDiffRefID, repo, reference, internalRefresh, prefix,
+                                         getMore, loading, nextPage, setAfterUpdated, onNavigate, onRevert}) => {
+    const [showComingSoonModal, setShowComingSoonModal] = useState(false);
+    const enableDeltaDiff = JSON.parse(localStorage.getItem(`enable_delta_diff`));
+    const sendDeltaDiffStats = async () => {
+        const deltaDiffStatEvents = [
+            {
+                "class": "experimental-feature",
+                "name": "delta-diff",
+                "count": 1,
+            }
+        ];
+        await statistics.postStatsEvents(deltaDiffStatEvents);
+    }
+
+    return <>
+        <div className="tree-container">
+            {(results.length === 0) ? <Alert variant="info">No changes</Alert> : (
+                <>
+                    {!enableDeltaDiff ?
+                        <>
+                        {showExperimentalDeltaDiffButton
+                            ? <>
+                            <ComingSoonModal display={showComingSoonModal}
+                                             onCancel={() => setShowComingSoonModal(false)}>
+                                <div>lakeFS Delta Lake tables diff is under development</div>
+                            </ComingSoonModal>
+                            <ExperimentalOverlayTooltip>
+                                <Button className="action-bar"
+                                        variant="primary"
+                                        disabled={false}
+                                        onClick={async () => {
+                                            setShowComingSoonModal(true);
+                                            await sendDeltaDiffStats();
+                                        }}>
+                                    <DiffIcon/> Compare Delta Lake tables
+                                </Button>
+                            </ExperimentalOverlayTooltip>
+                            </>
+                            : ""}</>
+                        :
+                        <div className="mr-1 mb-2">
+                            <Alert variant={"info"}><InfoIcon/> You can now use lakeFS to compare Delta Lake tables</Alert>
+                        </div>
+                    }
+                    <Card>
+                        <Card.Header>
+                                <span className="float-start">
+                                    {(delimiter !== "") && uriNavigator}
+                                </span>
+                        </Card.Header>
+                        <Card.Body>
+                            <Table borderless size="sm">
+                                <tbody>
```

Comment:
Is this supposed to be a table? It looks like a table with a body containing a list of `TreeItem`s

Reply:
I second this 👆🏻
There's at least one case where `TreeItem` doesn't return a `tr`
If you want to make it more apparent, you could name it `TreeTableRow` or something like that. You can also change the `.jsx` to `.tsx` and TS will make it clear that whatever is returned from `TreeItem` has to be a `tr` or `tr`-adjacent.

Reply:
I agree this is not a real table, if I had to guess why this was implemented as a table I'd say it is here for creating an easy-to-organize grid. any other suggestions for react component I can use?

Reply:
@talSofer I don't have a problem with the table itself. There are other ways of building something similar, but you could make a case for using a table.  
My main issue is that `tbody` should include `tr` tags only at the top level, and in this case, `TreeItem` includes at least one path where it doesn't return a `tr`.  
As a side note, I'd suggest renaming `TreeItem` to something like `TreeItemRow` or `TreeTableRow`, so it's more obvious that it's intended to return something with a `tr` at the top level. Hope that makes more sense.

Reply:
thanks @eladlachmi!

> My main issue is that tbody should include tr tags only at the top level, and in this case, TreeItem includes at least one path where it doesn't return a tr.

So in that case, would you just remove the table body? in addition to renaming TreeItem

Reply:
@talSofer Probably not. I suggest making sure that `TreeItem` (or whatever its new name will be) always returns a component with a single (or maybe even multiple, if there's a reason to) `<tr>` at the top level.

You can't just remove `tbody`, because the issue cascades. `<table>` also can't have just any random child tags. If you want to get rid of the table completely, that's a bigger refactor, and I wouldn't go there right now if you ask me.

Reply:
got it, fixed